### DISCUSSION TITLE
[Question] Python API for Light source position adjustment

### DISCUSSION THREAD

**Discussion by junhyeokahn** [bulletphysics/bullet3#3596] [General]
Hello,
Is there a python API for changing light source in GUI?
I saw some related work for camera images but not for GUI light source.

**Comment by mbusy** [comment]
Hello, I don't think so, but that could indeed be a very nice addition

**Comment by madurner** [comment]
@junhyeokahn could you share the related work you mentioned :) thanks

**Comment by mbusy** [comment]
By the way,
> Is there a python API for changing light source in GUI?

This has been resolved in PR #2298

**Comment by junhyeokahn** [comment]
@maedmaex Hi, please see [this](https://github.com/junhyeokahn/PnC/blob/7ef3cc3163d4aa1f32d8b5271223c3dba5f5d1da/ReinforcementLearning/MyGym/WalkerBaseEnv.py#L120).
Note that this is modified based on [pybullet-gym](https://github.com/benelot/pybullet-gym).
